webrender/src/render_backend.rs

```diff
@@ -344,6 +321,7 @@ impl RenderBackend {
                         }
                         ApiMsg::GenerateFrame => {
                             let frame = profile_counters.total_time.profile(|| {
+                                self.build_scene();
```

Comment:
We should be able to remove the build_scene() from here, once it's re-added to the other API messages.